packages/mock-addon/preset.js

```diff
@@ -8,5 +8,5 @@ function managerEntries(entry = []) {
 
 module.exports = {
     managerEntries,
-    config,
+    previewAnnotations,
```

Comment:
This would be a breaking change for older storybook version, since the entry has or will change